changelogs/fragments/add_hwaddr_filter.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Add hwaddr filter plugin.
```

Minor quibble, but these can all be in the same file if you're merging them at once
```suggestion
  - Add hwaddr filter plugin.
  - Add macaddr filter plugin.
  - Add slaac filter plugin.
```

---

ack

---

@Qalthos in the old ipaddr filters both hwaddr and macaddr mapped as different filter that's why created separate filters https://github.com/ansible-collections/ansible.netcommon/blob/main/plugins/filter/ipaddr.py#L1163